index.html

```diff
@@ -55,22 +55,27 @@ <h1 class="name-intro">
             <div class="works-card1">
                 <div class="child-worksCard-info">
                     <img src="./images/SnapshootPortfolio.png" alt="snapshoot">
-                    <h2 class="prjct-title">TONIC</h2>
-                    <div class="prjct-info">
-                        <h6 class="canopy client-name">CANOPY</h6>
-                        <div class="dot-separator"></div>
-                        <h6 class="client-name">Back End Dev</h6>
-                        <div class="dot-separator"></div>
-                        <h6 class="client-name">2015</h6>
-                    </div>
-                    <p class="description">A daily selection of privately <br> personalized reads; no accounts or <br>
-                        sign-ups required.</p>
-                    <div class="pro-languages">
-                        <p class="html">html</p>
-                        <p class="css">css</p>
-                        <p class="javascript">javascript</p>
+
+                    <div class="card-description">
+                        <h2 class="prjct-title">TONIC</h2>
+                        <div class="prjct-info">
+                            <h6 class="canopy client-name">CANOPY</h6>
+                            <div class="dot-separator"></div>
+                            <h6 class="client-name">Back End Dev</h6>
+                            <div class="dot-separator"></div>
+                            <h6 class="client-name">2015</h6>
+                        </div>
+                        <p class="description">A daily selection of privately  personalized reads; no accounts or 
+                            sign-ups required.</p>
+                        <div class="pro-languages">
+                            <p class="html">html</p>
+                            <p class="css">css</p>
+                            <p class="javascript">javascript</p>
+                        </div>
```

Comment:
- [ ] Kindly place these technologies (HTML. CSS, and JavaScript)  in a `ul` tag and each of them an `li` for best semantic practices. Apply this to all project cards 👍

Reply:
- [ ] Nice Job, you have implemented the change ✔️